integration-test/application.yml

```diff
@@ -1,438 +0,0 @@
-spring:
-    freemarker:
-        checkTemplateLocation: false
-
-# The logfiles will be created in this directory, LOG_PATH system property will be set and can be used in logback.xml
-# http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-logging.html#boot-features-logging-file-output
-logging:
-    path: ${integrationtest.outputdir}
-
-integrationtest:
```

Comment:
These default values are required for e2e tests to work

Reply:
application.yml is part of the integration-test project

Reply:
also necessary for mock tests

Reply:
the file itself was committed by mistake. the correct one resides under src/main/resources